overlord/snapstate/storehelpers.go

```diff
@@ -292,6 +303,11 @@ func refreshCandidates(ctx context.Context, st *state.State, names []string, use
 		return nil, nil, nil, err
 	}
 
+	var systemSeedTime string
+	if err := st.Get("seed-time", &systemSeedTime); err != nil {
```

Comment:
in general we might get here before seed-time is set, in which case we need a clearer error, or maybe seed-time is bad choice for this

Reply:
Looks like I've hit this in spread tests:
```
2018-09-06 12:42:14 Error executing google:fedora-28-64:tests/main/interfaces-content : 
-----
+ echo 'When a snap declaring a content sharing plug is installed'
When a snap declaring a content sharing plug is installed
+ snap install --edge test-snapd-content-plug
error: cannot install "test-snapd-content-plug": cannot obtain store request
       seed: no state entry for key
-----
```
I'm wondering whether I was just lucky or there's something else in play here.

Reply:
ensure loop and API ops are fairly orthogonal, we can get the latter before having gone through part of the former. At this point I think some kind of dedicated value stored in state once if it's not there seems more appealing.

Reply:
Yeah, seems like there's no way around it. The last batch of commits replaces this with separate `request-salt` kept in state. It's generated on demand if there's none.